solutions/src/main/java/ru/sberbank/school/task08/state/GameObject.java

```diff
@@ -1,13 +1,24 @@
 package ru.sberbank.school.task08.state;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
 import java.util.Objects;
 
-public class GameObject implements InstantiatableEntity {
+public class GameObject implements InstantiatableEntity, Serializable {
     private final Type type;
     private final Status status;
     private long hitPoints;
 
-    public GameObject(Type type, Status status, long hitPoints) {
+    private GameObject() {
+        type = null;
+        status = null;
+    }
+    @JsonCreator
```

Comment:
&#39;CTOR_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-643602824>*